src/coreclr/jit/emitarm64.cpp

```diff
@@ -11589,28 +11589,28 @@ void emitter::emitIns_Call(EmitCallType          callType,
 
 /*****************************************************************************
  *
- *  Return an encoding for the specified 'V' register used in '21' thru '17' position.
+ *  Return an encoding for the specified 'V' register used in '20' thru '16' position.
  */
 
-/*static*/ emitter::code_t emitter::insEncodeReg_V_21_to_17(regNumber reg)
+/*static*/ emitter::code_t emitter::insEncodeReg_V_20_to_16(regNumber reg)
 {
     assert(isVectorRegister(reg));
     emitter::code_t ureg = (emitter::code_t)reg - (emitter::code_t)REG_V0;
     assert((ureg >= 0) && (ureg <= 32));
```

Comment:
This is unrelated to your PR, but:
```suggestion
    assert((ureg >= 0) && (ureg <= 31));
```